website/src/controllers/api-v3/members.js

```diff
@@ -9,8 +9,18 @@ import { model as Group } from '../../models/group';
 import { model as Challenge } from '../../models/challenge';
 import {
   NotFound,
+  NotAuthorized,
 } from '../../libs/api-v3/errors';
 import * as Tasks from '../../models/task';
+import { chatDefaults } from '../../models/group';
+import refPush from '../../../../common/script/libs/refPush.js';
```

Comment:
we've started precompiling the server and shared code so you should only import the main `shared` module and then access the submodules from it, like https://github.com/HabitRPG/habitrpg/blob/api-v3/website/src/controllers/api-v3/tasks.js#L400